src/KeyboardInputModal.js

```diff
@@ -232,13 +235,20 @@ class KeyboardInputModal extends React.Component<KeyboardInputModalProps, Keyboa
     }
 
     // Required to avoid a phantom state where we persist the defaults even after they are updated from local storage.
-    componentDidUpdate (prevProps) {
+    componentDidUpdate (prevProps: KeyboardInputModalProps) {
         if (prevProps.keyBindingsEnabled !== this.props.keyBindingsEnabled || prevProps.keyboardInputSchemeName !== this.props.keyboardInputSchemeName) {
             this.setState({
                 keyBindingsEnabled: this.props.keyBindingsEnabled,
                 keyboardInputSchemeName: this.props.keyboardInputSchemeName
             });
         }
+        if (prevProps.show !== this.props.show && this.props.show) {
+            const keyboardShortcutsToggle = document.getElementById('keyboardInputModal__toggle');
+            if (keyboardShortcutsToggle) {
+                keyboardShortcutsToggle.focus();
+            }
+
```

Comment:
This kind of demonstrates my point about making a common utility, but again, we should do it in another ticket.  Please add a TODO here.  Also, the extra blank line is here too.